x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.scss

```
@@ -25,4 +25,20 @@
     background-color: $euiPageBackgroundColor;
     padding: $euiSizeL;
   }
+  
+  .documentsSearchExperience__facet {
+    .euiCheckboxGroup__item {
+      // Adding text-overflow to euiCheckbox__label below
+      // causes approximately 3px of bottom margin. this corrects
+      // that.
+      margin-bottom: -3px;
+    }
+
+    .euiCheckbox__label {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      width: 100%;
+    }
```

[Optional, not a change request] Totally up to you, but you can also use [EUI's text truncate helper/mixin](https://github.com/elastic/eui/blob/master/src/global_styling/mixins/_typography.scss#L108-L115) here if you like:

```suggestion
    .euiCheckbox__label {
      @include euiTextTruncate;
    }
```

---

https://github.com/elastic/kibana/pull/88491/commits/c9f39798633a92ea5d9987ed33b5f94cd33784d0